app/Http/Transformers/Three/SignupTransformer.php

```diff
@@ -3,8 +3,9 @@
 namespace Rogue\Http\Transformers\Three;
 
 use Rogue\Models\Signup;
+use Rogue\Services\Registrar;
 use League\Fractal\TransformerAbstract;
-use Rogue\Http\Transformers\PostTransformer;
+use Rogue\Http\Transformers\UserTransformer;
```

Comment:
Looks like `PostTransformer` is still used in `includePosts`

Reply:
@katiecrane I don't think we need that though since it is in the same directory! (Using `\Transformers\Three\PostTransformer` now :))

Reply:
shouldn't we have a `Three/UserTransformer`?

Reply:
oh cool! nice @chloealee !

Reply:
@sbsmith86 we totally should! I initially didn't do a new `UserTransformer` because I thought it'd be exactly the same file, but actually now that I think about this more, I think we should do some logic with who can see what about the user in the new `UserTransformer` instead of in the `includeUser` block in the `SignupTransformer`. 

Is it safe to return a user's first name and hide everything else for non-admins/non-owners of the signup? And then for admins/owners of the signup, they can see everything (`first_name`, `last_name`, `birthdate`, `email`, `mobile`)?

Reply:
@chloealee That makes sense to me. Do we display any other user information  (in the gallery, for example) ? Maybe age?

In general we should keep all of the V3 files unique and separate (even if it is a dupe), it will make it easier to archive and re-arrange files later.

Reply:
I'm not sure how things are hooked up now, but ideally we can forward along the `Authorization` header when asking Northstar for user details & that'll handle showing/hiding fields.

Reply:
ooo interesting @DFurnes ! So when grabbing the user, will we need to change the [`asClient()`](https://github.com/DoSomething/rogue/blob/master/app/Services/Registrar.php#L33) part to grab user info based on `Authorization` header?

Reply:
Yeah, that potentially interferes with our ability to cache users (since what we cache depends on what a particular user can see) so it might not make sense to run through the Registrar.

Reply:
Longer-term, this is a great example of where we'd want to use an API gateway (like [the GraphQL prototype](https://graphql.dosomething.org/explore) I worked on last quarter), rather than having Rogue piece together entire responses.

Reply:
ah yes! This totally makes sense. Matt and I were chatting about GraphQL at my last checkin too and how it could be a great use case when we need to pull in a user's college (and college logo) for more Domino work. cc @katiecrane 

@DFurnes for now, is it ok to just do this logic in the new `UserTransformer`?

Reply:
@chloealee I'm confused about how that would help us because the colleges and college logos will live in Domino, unless the suggestion is that those live somewhere else?

Reply:
@katiecrane sorry for the confusion! I might have missed this but where does the college data live (where the user attends college)? We were just briefly chatting that if we'd have to ping NS for this info. for a user's college, GraphQL might be a nice way to just make one call instead of multiple API calls to get this info.

Reply:
@chloealee ahh yeah that would make sense, but I believe the college info will all just live in Domino

Reply:
@katiecrane ah got it! Makes sense! :)